packages/app/ui/divider/Divider.tsx

```diff
@@ -0,0 +1,15 @@
+"use client";
+
+import { twMerge } from "tailwind-merge";
+
+interface DividerProps {
+  className?: string;
+  vertical?: boolean;
+}
+
+export const Divider = ({ className, vertical = false }: DividerProps) =>
```

Comment:
I'll probably prefer to have to Divider components than a conditional one. @berteotti what's your thoughts on this?

Reply:
Agree, it's good to avoid conditional rendering on UI component. Having concrete components is a better approach.

Reply:
As I took a look, we don't need this UI component. We can use tailwind for this :)

Reply:
Even better. Let's remove this component then.

Reply:
Ack! Will do it